nx-X11/lib/src/xlibi18n/xim_trans.c

```diff
@@ -24,6 +24,10 @@
 # include "config.h"
 #endif
 
+#ifndef MAXHOSTNAMELEN
```

Comment:
I realize that, but since it used to work before we probably broke something. I'd rather have the actual problem fixed than treating symptoms.

If I remove both definitions from `x11_trans.c` and `xim_trans.c`, the build seems to work fine on a Debian unstable system. Is this problem only manifesting on Debian kFreeBSD or such?